src/components/calculator.js

```diff
@@ -0,0 +1,70 @@
+const Calculator = () => (
+  <div className="calculator">
+    <div className="screen " />
+    <div className="keyboad">
+      <button type="button" className="numpad">
+        AC
+      </button>
+      <button type="button" className="numpad">
+        +/-
+      </button>
+      <button type="button" className="numpad">
+        %
+      </button>
+      <button type="button" className="numpad operator">
+        +
+      </button>
+
+      <button type="button" className="numpad">
+        7
+      </button>
+      <button type="button" className="numpad">
+        8
+      </button>
+      <button type="button" className="numpad">
+        9
+      </button>
+      <button type="button" className="numpad operator">
+        x
+      </button>
+
+      <button type="button" className="numpad">
+        4
+      </button>
+      <button type="button" className="numpad">
+        5
+      </button>
+      <button type="button" className="numpad">
+        6
+      </button>
+      <button type="button" className="numpad operator">
+        -
+      </button>
+
+      <button type="button" className="numpad">
+        1
+      </button>
+      <button type="button" className="numpad">
+        2
+      </button>
+      <button type="button" className="numpad">
+        3
+      </button>
+      <button type="button" className="numpad operator">
+        ÷
+      </button>
+
+      <button type="button" className="numpad zero">
+        0
+      </button>
+      <button type="button" className="numpad">
+        .
+      </button>
+      <button type="button" className="numpad operator">
+        =
+      </button>
+    </div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- Awesome calculator :clap: 
- [ ] According to React's best practices, capitalizing the first letter of your components is a good one and that is why the requirement was also a bit specific about creating a `Calculator.js`. Kindly make this correction :pray: 
![image](https://github.com/Sami-Ullah-Tufail/Math-magician/assets/99715640/016591ac-e7e4-4c27-ab93-33366d387bf8)